editor/test/integration/save_test.clj

```diff
@@ -35,40 +34,15 @@
             [service.log :as log]
             [support.test-support :refer [spit-until-new-mtime touch-until-new-mtime with-clean-system]]
             [util.text-util :as text-util])
-  (:import [com.dynamo.gamesys.proto AtlasProto$Atlas]
-           [com.dynamo.gameobject.proto GameObject$CollectionDesc GameObject$PrototypeDesc]
-           [com.dynamo.gamesys.proto Gui$SceneDesc]
-           [com.dynamo.gamesys.proto Label$LabelDesc]
-           [com.dynamo.gamesys.proto ModelProto$ModelDesc]
-           [com.dynamo.particle.proto Particle$ParticleFX]
-           [com.dynamo.render.proto Font$FontDesc]
-           [com.dynamo.gamesys.proto Tile$TileGrid Tile$TileSet]
-           [java.io StringReader File]
+  (:import [java.io StringReader File]
            [org.apache.commons.io FileUtils]
            [org.eclipse.jgit.api Git ResetCommand$ResetType]))
 
-(def ^:private ext->proto {"atlas" AtlasProto$Atlas
-                           "font" Font$FontDesc
-                           "go" GameObject$PrototypeDesc
-                           "collection" GameObject$CollectionDesc
-                           "gui" Gui$SceneDesc
-                           "label" Label$LabelDesc
-                           "model" ModelProto$ModelDesc
-                           "particlefx" Particle$ParticleFX
-                           "tilegrid" Tile$TileGrid
-                           "tilemap" Tile$TileGrid
-                           "tileset" Tile$TileSet
-                           "tilesource" Tile$TileSet})
-
```

Comment:
We can use the `:read-raw-fn` of the `resource-type` map instead of this hard-coded mapping now. 🥳